plugwise/helper.py

```diff
@@ -0,0 +1,919 @@
+"""
+Use of this source code is governed by the MIT license found in the LICENSE file.
+
+Plugwise protocol helpers
+"""
+import asyncio
+import datetime as dt
+import logging
+
+import async_timeout
+from dateutil.parser import parse
+from defusedxml import ElementTree as etree
+
+# Time related
+import pytz
+
+from .constants import (
+    APPLIANCES,
+    ATTR_NAME,
+    ATTR_TYPE,
+    ATTR_UNIT_OF_MEASUREMENT,
+    DEVICE_MEASUREMENTS,
+    DOMAIN_OBJECTS,
+    ENERGY_KILO_WATT_HOUR,
+    ENERGY_WATT_HOUR,
+    HEATER_CENTRAL_MEASUREMENTS,
+    HOME_MEASUREMENTS,
+    LOCATIONS,
+    MODULES,
```

Comment:
MODULES is unused

Reply:
otherwise didn't fully (again) review since we've already checked it twice (firstly in smile.py then with us both checking over (than called `helpers.py`) :)